jmaqs-selenium/src/main/java/com/magenic/jmaqs/selenium/BaseSeleniumPageModel.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2020 (C) Magenic, All rights Reserved
+ */
+
+package com.magenic.jmaqs.selenium;
+
+import com.magenic.jmaqs.selenium.factories.UIWaitFactory;
+import com.magenic.jmaqs.utilities.logging.Logger;
+import com.magenic.jmaqs.utilities.performance.PerfTimerCollection;
+import java.util.HashMap;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+/**
+ * The type Base selenium page model.
+ */
+public abstract class BaseSeleniumPageModel {
+
+  private final HashMap<String, LazyWebElement> lazyElementStore;
```

Comment:
Will sonar yell at us for not having javadocs on instance variables and methods? There are a few missing in this class if so.

Reply:
Private fields are exempt from it.  Same with Checkstyle.